### PR TITLE
Fixes another MacSpace stuff

### PR DESCRIPTION
**Pull request by OceanFish1**
## About The Pull Request

Who could guess that there's a water turf in the middle of the room that wasn't var edited by a funny mapper and caused all the air being fucked up even with tiny fans.

## Why It's Good For The Game

dude

## Changelog
:cl:
fix: MacSpace should be visitable now without internals required
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/Fast_Food.dmm [hunk 1]
```diff
@@ -598,7 +587,10 @@
 "ck" = (
 /obj/machinery/light/floor,
 /obj/effect/decal/cleanable/dirt/dust,
-/turf/open/water,
+/turf/open/water{
+	initial_gas_mix = "o2=22;n2=82;TEMP=293.15";
+	planetary_atmos = 0
+	},
```
Comment (park66665) [inline _maps/RandomRuins/SpaceRuins/Fast_Food.dmm]: Making a subtype of `turf/open/water` with these attributes is a standard way to solve this problem. Append the resulting subtype to code/game/turfs/open/water.dm

**Review by park66665 — CHANGES_REQUESTED** [review]

File: _maps/RandomRuins/SpaceRuins/Fast_Food.dmm (hunk 1 above)
Comment (OceanFish1) [reply _maps/RandomRuins/SpaceRuins/Fast_Food.dmm]: making a whole new subtype just to get it used on one ruin? ok

File: code/game/turfs/open/water.dm [hunk 2]
```diff
@@ -15,3 +15,7 @@
 	barefootstep = FOOTSTEP_WATER
 	clawfootstep = FOOTSTEP_WATER
 	heavyfootstep = FOOTSTEP_WATER
+
+/turf/open/water/air
+	initial_gas_mix = "o2=22;n2=82;TEMP=293.15"
+	planetary_atmos = 0
```
Comment (park66665) [inline code/game/turfs/open/water.dm]:
```suggestion
	planetary_atmos = FALSE
```

File: _maps/RandomRuins/SpaceRuins/Fast_Food.dmm (hunk 1 above)
Comment (park66665) [reply _maps/RandomRuins/SpaceRuins/Fast_Food.dmm]: Most people don't realize, but varedit is ~~technically~~ a sin.

File: code/game/turfs/open/water.dm [hunk 3]
```diff
@@ -15,3 +15,7 @@
 	barefootstep = FOOTSTEP_WATER
 	clawfootstep = FOOTSTEP_WATER
 	heavyfootstep = FOOTSTEP_WATER
+
+/turf/open/water/air
+	initial_gas_mix = "o2=22;n2=82;TEMP=293.15"
```
Comment (park66665) [inline code/game/turfs/open/water.dm]:
```suggestion
	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
```

**Review by park66665 — APPROVED** [review]
All clear. This PR is good to go.

**Pull request merged**